custom/rch/models.py

```diff
@@ -0,0 +1,159 @@
+import datetime
+from django.db import models
+from django.conf import settings
+from custom.rch.utils import (
+    fetch_beneficiaries_records,
+    find_matching_cas_record_id,
+)
+from jsonfield.fields import JSONField
+from custom.rch.const import (
+    MOTHER_RECORD_TYPE,
+    CHILD_RECORD_TYPE,
+    STATE_DISTRICT_MAPPING,
+    RCH_RECORD_TYPE_MAPPING,
+)
+
+
+class RCHRecord(models.Model):
+    cas_case_id = models.CharField(null=True, max_length=255)  # ICDS-CAS case that was found as a match
+    details = JSONField(default=dict)  # all details received from RCH for this beneficiary
+    district_id = models.PositiveSmallIntegerField(null=True)
+    state_id = models.PositiveSmallIntegerField(null=True)
+    village_id = models.IntegerField(null=True)
+    village_name = models.CharField(null=True, max_length=255)
+    name = models.CharField(null=True, max_length=255)
+    aadhar_num = models.BigIntegerField(null=True)
+    dob = models.DateTimeField(null=True)
+    rch_id = models.BigIntegerField(null=True)
```

Comment:
should this be unique?

Reply:
Also it looks like you're filtering on it so you'll almost certainly want an index

Reply:
Yes.
Yes.